### PR TITLE
Sync Akka & Lagom projects with the latest releases

### PR DESCRIPTION
**Pull request by marcospereira**
Includes recent changes made in Akka Plataform Guide and Lagom samples.﻿

### CODE REVIEW

File: akka/shopping-cart-java/docker-compose.yml [hunk 1]
```diff
@@ -18,20 +18,38 @@ services:
...
+      - kafka
+    volumes:
+      - ./kafka-1-logs:/tmp/kafka-logs
+
+  wait-for-kafka:
+    image: waisbrot/wait
+    links:
+      - kafka
+    environment:
+      - TARGETS=kafka:9092
```
Comment (ignasi35) [inline akka/shopping-cart-java/docker-compose.yml:55]: Nice to see this spreading. It is not water-proof but it helps a bit.

File: akka/shopping-cart-java/kubernetes/shopping-cart-service-cr.yml [hunk 2]
```diff
@@ -22,4 +22,3 @@ spec:
     credentialsSecret: shopping-cart-service-jdbc-secret
   kafka:
     credentialsSecret: shopping-cart-service-kafka-secret  # <2>
-  prometheusPort: 9001
```
Comment (ignasi35) [inline akka/shopping-cart-java/kubernetes/shopping-cart-service-cr.yml:25]: Did you mean to remove this?

File: akka/shopping-cart-java/pom.xml [hunk 3]
```diff
@@ -119,7 +118,7 @@
         <dependency>
             <groupId>org.hibernate</groupId>
             <artifactId>hibernate-entitymanager</artifactId>
-            <version>5.4.30.Final</version>
+            <version>5.4.13.Final</version>
```
Comment (ignasi35) [inline akka/shopping-cart-java/pom.xml:121]:
```suggestion
            <version>5.4.31.Final</version>
```
Was this a typo?

**Review by ignasi35 — APPROVED** [review]
I was doing a manual review but I suspect you `rsync`'d it, correct?

**Comment by marcospereira** [comment]
I will open a new one.

**Pull request closed without merging**